src/Command/CreateCommand.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace TrendingRepos\Command;
+
+use Symfony\Component\Console\Command\Command;
+
+abstract class CreateCommand extends Command
```

Comment:
This is not proper usage of an abstract class, it acts as an interface and only forces all child classes to implement some methods. In the end, you still have a lot of code duplication in both `Create*Command`. Could you think of design improvement here?

Reply:
I had most functions now in `CreateCommand.php` i wanted to make `generateFileContent` abstract too but didn't want to make it public too.. so is it better this way ? Or switch to pulblic ?

Reply:
Think as  im calling `generateFileContent` within the abstract class i should make it public and add abstract function there .. that makes more sense for me .. i will do that :D

Reply:
Links <3 
Good to know about this pattern 

By the way, I swear i thought i tried to keep it protected and some error appeared over there on 'vscode' !  maybe there was something else and i guessed it cannot be protected :neutral_face: